### PR TITLE
Add gzip compression and bump version on jetty

### PR DESCRIPTION
**Pull request by erebe**
Mainly the same as #95 but without conflicts

mvn integration-test is ok and I use the standalone http server with gzip activated in production

### CODE REVIEW

File: jmx_prometheus_javaagent/pom.xml [hunk 1]
```diff
@@ -33,14 +33,19 @@
...
-      <version>8.1.7.v20120910</version>
+      <version>9.4.3.v20170317</version>
     </dependency>
     <dependency>
       <groupId>junit</groupId>
       <artifactId>junit</artifactId>
       <version>4.12</version>
       <scope>test</scope>
     </dependency>
+      <dependency>
```
Comment (brian-brazil) [inline jmx_prometheus_javaagent/pom.xml]: This should go above the test dependency

Comment (erebe) [inline jmx_prometheus_javaagent/pom.xml]: done

**Comment by erebe** [comment]
Pom updated

**Comment by brian-brazil** [comment]
Thanks!

**Pull request merged**